### DISCUSSION TITLE
File Manager Virtual Xposed Problem

### DISCUSSION THREAD

**Discussion by NoellyMiyaaamuyo** [android-hacker/VAExposed#1162] [Q&A]
Hello, I'm new here and wanted to ask questions about something.
When I installed "AmazeFileManager" in Virtual Xposed it always directed at my phone real directory, not the one in the Virtual Xposed. Why and How to fix this?

How did I managed to install file manager? Well I install it from *link* and install it in Virtual Xposed

> https://github.com/TeamAmaze/AmazeFileManager/releases?page=1

I wan't to edit my game data on virtual xposed but can't because of it.

(I been researching and looking at forums or issues still can't find the answer and when i tried to redirect it to

> /data/data/io.va.exposed64

Still not working, and there's the storage permission error even though i already granted it)

![Directed At Real Phone Directory](https://github.com/android-hacker/VirtualXposed/assets/135057341/c882fcd7-599c-4dfb-ace3-53511727092a)

![Grant Permission Problem](https://github.com/android-hacker/VirtualXposed/assets/135057341/9b570372-ce96-471b-9e14-a579effb6c56)

![Grant Permission Problem 2 - Can't Turn On Though](https://github.com/android-hacker/VirtualXposed/assets/135057341/7033b4db-7dcb-4920-960c-58c8a568e581)

![Can't Go To](https://github.com/android-hacker/VirtualXposed/assets/135057341/2f76e53e-a491-4a67-8176-a33e22f88224)

Directory reference to
#293

**Comment by didiaodanding** [comment]
您好，您的邮件已收到，谢谢。                                    --------李峰
